src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -186,7 +199,7 @@ create_distributed_table(PG_FUNCTION_ARGS)
 
 		ConvertToDistributedTable(relationId, distributionColumnName,
 								  distributionMethod, REPLICATION_MODEL_COORDINATOR,
-								  INVALID_COLOCATION_ID);
+								  INVALID_COLOCATION_ID, allowEmpty);
```

Comment:
Figured it out. `allowEmpty` says that empty is permitted, but doesn't say anything about whether non-empty is permitted. It could be allowed too, by that naming.

`requireEmpty` is probably what we mean. The boolean values must be flipped, but it makes the code make a lot more sense from my perspective.